script/generate_incremental_table

```diff
@@ -101,7 +101,9 @@ def query(day, entrypoint, destination_table, query_options, query_file):
 
 
 def main():
-    args = parser.parse_args()
+    args, unknown_args = parser.parse_known_args()
```

Comment:
This allows the user to pass `--dataset`, etc.

Reply:
they would be included in `query_options` if preceded with a `--` option, but I love that this takes away the restriction on where they are placed. 🎉